### PR TITLE
BRAVO-2191-fix-1155 partly purchase

### PR DESCRIPTION
**Pull request by SKonstantinS**
Add fix which allow purchase 1155 in Rarible orders partly (for example 2 from 10).
Ask for review

### CODE REVIEW

File: exchange-wrapper/contracts/ExchangeWrapper.sol [hunk 1]
```diff
@@ -101,12 +101,18 @@ contract ExchangeWrapper is OwnableUpgradeable {
...
         LibOrder.Order memory buyerOrder;
         buyerOrder.maker = address(this);
         buyerOrder.makeAsset = sellOrder.takeAsset;
-        buyerOrder.takeAsset = sellOrder.makeAsset;
+        buyerOrder.takeAsset.assetType = sellOrder.makeAsset.assetType;
+        if (buyerOrder.takeAsset.assetType.assetClass == LibAsset.ERC1155_ASSET_CLASS) {
+            buyerOrder.takeAsset.value = purchaseAmount; //for ERC1155 set the exact amount for purchase
+        } else {
+            buyerOrder.takeAsset.value = sellOrder.makeAsset.value; // for ERC721 is always == 1
+        }
```
Comment (disenotov) [inline exchange-wrapper/contracts/ExchangeWrapper.sol:115]: why do we need this if? if purchaseAmount is always present, we can assume that for ERC721 it's going to be 1. or not?

File: exchange-wrapper/test/v2/ExchangeBulkV2.rarible.test.js [hunk 2]
```diff
@@ -371,6 +468,71 @@ contract("ExchangeBulkV2, sellerFee + buyerFee =  6%,", accounts => {
...
+    		    verifyBalanceChange(seller3, -100, async () =>
+    			    verifyBalanceChange(feeRecipienterUP, -36, () =>
+    				    bulkExchange.bulkPurchase([tradeData1, tradeData2, tradeData3], feesUP, { from: buyer, value: 400, gasPrice: 0 })
+    				  )
+    				)
+    			)
+    		)
+    	);
+//      const tx = await bulkExchange.bulkPurchase([tradeData1, tradeData2, tradeData3], feesUP, { from: buyer, value: 400, gasPrice: 0 });
+//      console.log("Bulk, by bulkPurchase ERC721<->ETH (num = 3), Gas consumption :",tx.receipt.gasUsed);
```
Comment (disenotov) [inline exchange-wrapper/test/v2/ExchangeBulkV2.rarible.test.js:527]: unnecessary comments

File: exchange-wrapper/contracts/ExchangeWrapper.sol (hunk 1 above)
Comment (SKonstantinS) [reply exchange-wrapper/contracts/ExchangeWrapper.sol:115]: I agree, better to use simple:
buyerOrder.takeAsset.value = purchaseAmount;
without any If, "if" don`t save from error if parameter  **purchaseAmount** is incorrect

File: exchange-wrapper/test/v2/ExchangeBulkV2.rarible.test.js (hunk 2 above)
Comment (SKonstantinS) [reply exchange-wrapper/test/v2/ExchangeBulkV2.rarible.test.js:527]: agree delete

**Pull request merged**